### ISSUE TITLE
DuetDisplay Request

### ISSUE THREAD

**Issue by dtantono** [microsoft/winget-pkgs#22492]
PackageIdentifier:   DuetDisplay
PackageVersion:      1.9.8
PackageLocale:       en-US
Publisher:           duet
PackageName:         duetdisplay
License:             Proprietary
ShortDescription:    turn tablet/smartphone to become a secondary monitor
Installers: 
 - Architecture:     x64
   InstallerType:    exe
   InstallerUrl:     https://duet.nyc3.cdn.digitaloceanspaces.com/Windows/1_9/DuetSetup-1-9-8-2.exe
ManifestType: singleton
ManifestVersion: 1.0.0

**Comment by jedieaston** [comment]
I don't think we can do this with the current validation workflow (as we need to verify the manifest inside a virtual machine):

<img width="810" alt="image" src="https://user-images.githubusercontent.com/21368066/127147990-00eb9ca1-0b6e-438a-8ce8-0b187acde75e.png">

**Comment by KevinLaMS** [comment]
As mentioned, we are blocked on this as validation is done on VMs.
